src/test/java/be/seeseemelk/mockbukkit/scoreboard/TeamMockTest.java

```diff
@@ -40,58 +42,60 @@ public void setUp() throws Exception
 		team = new TeamMock("Test", board);
 	}
 
-	@After
+	@AfterEach
 	public void tearDown() throws Exception
 	{
-		MockBukkit.unload();
+		MockBukkit.unmock();
 	}
 
 	@Test
-	public void getName()
+	void getName()
 	{
 		assertEquals("Test", team.getName());
 	}
 
 	@Test
-	public void getPrefix()
+	void getPrefix()
 	{
 		assertNull(team.getPrefix());
 		team.setPrefix("THIS");
 		assertEquals("THIS", team.getPrefix());
 	}
 
 	@Test
-	public void getColor()
+	void getColor()
 	{
 		assertNull(team.getColor());
 		team.setColor(ChatColor.AQUA);
 		assertEquals(ChatColor.AQUA, team.getColor());
 	}
 
 	@Test
-	public void allowFriendlyFire()
+	void allowFriendlyFire()
 	{
 		assertFalse(team.allowFriendlyFire());
 		team.setAllowFriendlyFire(true);
 		assertTrue(team.allowFriendlyFire());
 	}
 
 	@Test
-	public void canSeeFriendlyInvisibles()
+	void canSeeFriendlyInvisibles()
 	{
 		assertTrue(team.canSeeFriendlyInvisibles());
 		team.setCanSeeFriendlyInvisibles(false);
 		assertFalse(team.canSeeFriendlyInvisibles());
 	}
 
+	@SuppressWarnings("deprecation")
 	@Test
-	public void getNameTagVisibility()
+	void getNameTagVisibility()
 	{
-		assertEquals(NameTagVisibility.ALWAYS, team.getNameTagVisibility());
+		assertEquals(org.bukkit.scoreboard.NameTagVisibility.ALWAYS, team.getNameTagVisibility());
```

Comment:
Why is the FQN used here instead of an import?

Reply:
Because `NameTagVisibility` is a deprecated enum.
I rather suppress deprecation warnings for a test instead of for a whole class because of an import.
Can change that if you want of course.